static/app/views/dashboardsV2/widgetCard/chart.tsx

```diff
@@ -142,7 +147,42 @@ class WidgetCardChart extends React.Component<WidgetCardChartProps> {
 
       const rendered = fieldRenderer(dataRow);
 
-      return <BigNumber key={`big_number:${result.title}`}>{rendered}</BigNumber>;
+      if (!!!organization.features.includes('dashboard-grid-layout')) {
+        return <BigNumber key={`big_number:${result.title}`}>{rendered}</BigNumber>;
+      }
+
+      const {widget} = this.props;
+      const widthToHeightRatio =
+        widget.layout?.w && widget.layout?.h ? widget.layout.w / widget.layout.h : 1;
+
+      const h = BIG_NUMBER_WIDGET_DEFAULT_HEIGHT;
+
+      // heuristics to maintain an aspect ratio that works
+      // most of the time.
+      const w =
+        widget.layout?.w && widget.layout?.h
+          ? widthToHeightRatio * 400
+          : BIG_NUMBER_WIDGET_DEFAULT_WIDTH;
+
+      const fontSize =
+        widthToHeightRatio < 1
```

Comment:
if widget is taller than it is wider, we shrink the font size based on the ratio